pkg/block/fetcher_test.go

```diff
@@ -429,7 +436,8 @@ func TestLabelShardedMetaFilter_Filter_Hashmod(t *testing.T) {
 			deleted := len(input) - len(expected)
 
 			synced := promauto.With(nil).NewGaugeVec(prometheus.GaugeOpts{}, []string{"state"})
-			f.Filter(input, synced, false)
+			err := f.Filter(ctx, input, synced, false)
```

Comment:
ditto and below

Reply:
yes, have made this change at other places as well.